dapp/src/components/common/withBridge/index.jsx

```diff
@@ -23,70 +25,136 @@ export default function withBridge(WrappedComponent) {
     const [blockNumber, setBlockNumber] = useState()
     const [confirmationsLimit, setConfirmationsLimit] = useState()
     const [delay, setDelay] = useState(null)
-    const [relayEvent, setRelayEvent] = useState(null)
+    const [newTokenRegisteredDelay, setNewTokenRegisteredDelay] = useState(null)
     const { accountAddress } = network
-    const { web3Context } = network
-    const { tokenContext } = dashboard
+    const { tokenContext, _web3Foreign } = dashboard
     const { web3: web3Home } = tokenContext
-    const { web3: web3Foreign } = web3Context
-    const { foreignNetwork, bridgeDirection } = dashboard
+    const { foreignNetwork } = dashboard
     const bridge = dashboard?.bridgeStatus?.from?.bridge
+    const balanceKey = dashboard?.bridgeStatus?.from?.balanceKey
+    const fromBalanceKey = dashboard?.bridgeStatus?.from?.balanceKey
```

Comment:
balanceKey and fromBalanceKey are the same value

Reply:
Thnaks

Reply:
Thanks